yara-x.yaml

```diff
@@ -23,9 +25,9 @@ pipeline:
       expected-commit: d2b8358879c009f41f0e14847681f2f8dbe624cf
       tag: v${{package.version}}
 
-  - runs: cargo install cargo-c --features=vendored-openssl --root $HOME/.cargo
+  - runs: cargo install cargo-c --features=vendored-openssl --root $HOME/.cargo --target-dir $HOME/.cargo
 
-  - runs: cargo cinstall -p yara-x-capi --release --prefix=/usr --pkgconfigdir=/usr/lib/pkgconfig --includedir=/usr/include --libdir=/usr/lib
+  - runs: RUSTFLAGS="-C opt-level=3" cargo cinstall -p yara-x-capi --release --prefix=/usr --pkgconfigdir=/usr/lib/pkgconfig --includedir=/usr/include --libdir=/usr/lib
```

Comment:
Nit: does it make sense to set this in the environment block?

Reply:
Good catch -- moved in [`7b5fe52` (#38392)](https://github.com/wolfi-dev/os/pull/38392/commits/7b5fe5244ff1b64a8c485cb838b1bc93b608f393).